ts/output/common/OutputJax.ts

```diff
@@ -545,6 +545,8 @@ export abstract class CommonOutputJax<
    *
    * @param {N} text         The text element to measure
    * @param {string} chars   The string contained in the text node
+   * @param {string} variant     The variant for the text
+   * @param {CssFontData} font   The style cssText string containing the font information
```

Comment:
This is not a string, but a triple `[string, boolean, boolean]` indicating the CSS font to use for the given variant, and wither the font is italic or bold (the two booleans).  Perhaps `The family, italic, and bold data for explicit fonts` would be best.